examples/quickstart/tutorial/hadoop/docker/Dockerfile

```diff
@@ -14,14 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Creates pseudo distributed hadoop 2.8.3 with java 8
-#
-# Modified from the SequenceIQ Dockerfiles at https://github.com/sequenceiq/hadoop-docker
-#
-# docker build -t druid-hadoop-demo:2.8.3 .
```

Comment:
for licensing reasons we should leave the code attribution to credit the source of this code